simulation/simulation.py

```diff
@@ -56,23 +59,35 @@ def __init__(self, no_spaces, min_spaces_per_lot, max_spaces_per_lot, no_cars,
             name += 1
 
         for i in range(self.no_cars):
-            start_time = 5  # TODO atm, delay until after spaces created, later add handling for no spaces
+            start_time = 1
             p = self.point_to_location(self.random_lot.randint(0, width), self.random_lot.randint(0, height))
             coro = car_routine(start_time, p, self)
             self.car_tasks.append(asyncio.ensure_future(coro))
 
-        self.tasks = self.space_tasks + self.car_tasks
+        rogue_start = 1
+        for i in range(self.no_rogues):
+            if random.randint(1, 2) == 1:
+                locx = random.choice([0, width])
+                locy = random.randint(0, height)
+            else:
+                locx = random.randint(0, width)
+                locy = random.choice([0, height])
+            loc = self.point_to_location(locx, locy)
+            dest = self.point_to_location(self.random_lot.randint(0, width), self.random_lot.randint(0, height))
+            self.rogue_tasks.append(asyncio.ensure_future(rogue_routine(rogue_start + i/10, loc, dest, self)))
+
+        self.tasks = self.space_tasks + self.car_tasks + self.rogue_tasks
         self.run_task = None
 
     def point_to_location(self, x: float, y: float) -> wsmodels.Location:
         """Assuming (0, 0) x/y maps to Location(0, 0), compute the Location for an arbitrary x, y point
         """
-        return wsmodels.Location(x / SCALE, y / SCALE)
+        return wsmodels.Location(x / (SCALE), y / (SCALE))
 
     def loc_to_point(self, loc: wsmodels.Location):
         """Assuming (0, 0) x/y maps to Location(0, 0), compute the Location for an arbitrary x, y point
         """
-        return (loc.longitude * SCALE, loc.latitude * SCALE)
+        return (loc.latitude * (SCALE), loc.longitude * (SCALE))
```

Comment:
Hmm, this is backwards isn't it? Latitude is north/south measurement, and I took that to be the y-axis.